haiku-libs/haikuwebkit/haikuwebkit-1.2.0.recipe

```diff
@@ -33,8 +31,14 @@ REQUIRES="
 	lib:libz$secondaryArchSuffix
 	"
 
-BUILD_REQUIRES="
-	haiku${secondaryArchSuffix}_devel
+PROVIDES_devel="haikuwebkit${secondaryArchSuffix}_devel = $portVersion
+	devel:libjavascriptcore$secondaryArchSuffix = $portVersion
+	devel:libwebcore$secondaryArchSuffix = $portVersion
+	devel:libwebkit$secondaryArchSuffix = $portVersion
+	devel:libwtf$secondaryArchSuffix = $portVersion"
+REQUIRES_devel="haikuwebkit$secondaryArchSuffix == $portVersion base"
```

Comment:
All PROVIDES*/*REQUIRES*/*FILES* get the extra line at the beginning and end, as I said before. So this should be 
```
PROVIDES_devel="
	haikuwebkit${secondaryArchSuffix}_devel = $portVersion
	devel:libjavascriptcore$secondaryArchSuffix = $portVersion
	devel:libwebcore$secondaryArchSuffix = $portVersion
	devel:libwebkit$secondaryArchSuffix = $portVersion
	devel:libwtf$secondaryArchSuffix = $portVersion
	"
```
Please fix these.